doc/expressionLanguage.md

```diff
@@ -17,6 +17,39 @@ The IoTAgent Library provides an expression language for measurement transformat
 information coming from the South Bound APIs to the information reported to the Context Broker. Expressions in this
 language can be configured for provisioned attributes as explained in the Device Provisioning API section in the
 main README.md.
+As an alternative, the IoTAgent Library supports as well [JEXL](https://github.com/TomFrost/jexl).
+To use JEXL, you will need to either configure it as default
+language using the `defaultExpressionLanguage` field to `jexl` or configuring
+the usage of JEXL as expression language for a given device:
+
+```
+{
+   "devices":[
+      {
+         "device_id":"45",
+         "protocol":"GENERIC_PROTO",
+         "entity_name":"WasteContainer:WC45",
+         "entity_type":"WasteContainer",
+         "expressionLanguage": "jexl",
+         "attributes":[
+            {
+               "name":"location",
+               "type":"geo:point",
+               "expression": "..."
+            },
+            {
+               "name":"fillingLevel",
+               "type":"Number",
+               "expression": "..."
+            }
+         ]
+      }
+   ]
+}
+```
+
+In this section we provide only details for the native IoTAgent Library 
+expression language. JEXL language details are discussed [here](https://github.com/TomFrost/jexl#all-the-details).
```

Comment:
It would be desirable to include in the PR a .md showing some examples of JEXL expressions. For instance, the same examples included in expressionLanguage.md or maybe some cases based in the ones of issue https://github.com/telefonicaid/iotagent-node-lib/issues/687, to show how the JEXL syntax improves the legacy one.

What do you think? It would be feasible to do that? It will be definitively very useful for people wanting to use this new outstating functionality.

Reply:
I think some example is important, indeed. Depending on your timeline to have this supported, it may be better to have a separate PR. (I won't have time before end of Jan probably)

Reply:
End of January is fine with me.

The rest of comments (minor ones) could be worked out in the meanwhile. We can provide a "LGTM at commit ..., pending on including examples documentation" when everything else get solved.

Reply:
Regarding examples to include, a good one would be how to use JEXL to build a (non-trivial) GeoJSON attribute value. That is not possible with the current expression language so it would show definetively the advantages of the new language :)

Reply:
i think this is covered in the example, right?

Reply:
Right, thanks. NTC.